docs/_instructions/connection-settings.md

```diff
@@ -97,4 +102,16 @@ In maven, the servers are identified by an id; when you define a repository you
 | `verify`          |  `true`      | `false | true` | Enable/disable the verification of the host name against a certificate for HTTPS |
 | `trust`           |              |                | Provide paths to certificate that provide trus to the host certificate |
 
-	
+### oAuth2 authentication
+
+It also supports Bearer (OAuth2) authentication. If the <server> configuration has only a password and no username, then Bearer authentication is in effect with the password used as the token.
+
+	-connection-settings: server;id="https://*.server.com";password="oauth2token"
+
+will cause
+
+Authorization: Bearer oauth2token
```

Comment:
You should start this line with 4 spaces.

Reply:
Good Point. I missed that.